jdat_notebooks/NIRSpec_MAST_Query/NIRSpec_MAST_Query.ipynb

```diff
@@ -0,0 +1,801 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MAST Query Tutorial for NIRSpec Users"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Learning Goals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By the end of this tutorial, you will:\n",
+    "\n",
+    "    1) Be able to perform MAST queries for NIRSpec data \n",
+    "    2) Understand MAST search options for NIRSpec MOS, IFU, and Fixed Slit data\n",
+    "    3) Filter MAST queries by various parameters\n",
+    "    4) Perform potential duplication checks for your targets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "MAST (Barbara A. Mikulski Archive for Space Telescopes) provides a means of retrieving data through a variety of methods. The [MAST Portal](https://outerspace.stsci.edu/display/MASTDOCS/Portal+Guide) allows users to interactively search for particular observations and filter by keywords. An in-depth [tutorial](https://mast.stsci.edu/portal/Mashup/Clients/Mast/data/html/MastHelp.html) is available to aid users in accessing MAST in this manner. In some scenarios, particularly with large queries, it may be more efficient to use [Astroquery](https://astroquery.readthedocs.io/en/latest/mast/mast.html). Some broad capabilities including positional queries, metadata queries, and catalog queries are described in the documentation. General documentation on the [MAST API](https://mast.stsci.edu/api/v0/) is also available.\n",
```

Comment:
```suggestion
    "MAST (Barbara A. Mikulski Archive for Space Telescopes) provides a means of retrieving data through a variety of methods. The [MAST Portal](https://outerspace.stsci.edu/display/MASTDOCS/Portal+Guide) allows users to interactively search for particular observations and filter by keywords. An in-depth [tutorial](https://outerspace.stsci.edu/display/MASTDOCS/Portal+Guide) is available to aid users in accessing MAST in this manner. In some scenarios, particularly with large queries, it may be more efficient to use [Astroquery](https://astroquery.readthedocs.io/en/latest/mast/mast.html). Some broad capabilities including positional queries, metadata queries, and catalog queries are described in the documentation. General documentation on the [MAST API](https://mast.stsci.edu/api/v0/) is also available.\n",
```